docs/pages/database-access/guides/redis-aws.mdx

```diff
@@ -0,0 +1,235 @@
+ACL](https://redis.io/docs/manual/security/acl/) for authentication with Redis
+and let Teleport manage the users. The Database Service constantly rotates
+Teleport managed user's password, saves the password in AWS Secrets Manager,
+and automatically sends `AUTH` with the saved password when connecting the
+client to the Redis server.
+
+To enable Redis ACL, please see [Authenticating users with Role-Based Access
+Control for
+ElastiCache](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Clusters.RBAC.html).
+
+Once an ElastiCache user is created with the desired access, add an
+AWS resource tag `teleport.dev/managed` with the value `true` to this user. The
+Database Service will automatically discover this user if it is associated with
+a registered database. Keep in mind that it may take the Database Service some
+time (up to 20 minutes) to discover this user once the tag is added.
+
+<Admonition type="tip">
+It is not required for all users in the same ElastiCache user group to be
+managed by Teleport. Just the tag users that are desired to be used for
+Teleport database access. You can still have other ElastiCache users for other
+usages like programming access.
+</Admonition>
+
+## Step 7/7. Connect
+
+Once the Database Service has started and joined the cluster, log in to see the
+registered databases:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name                        Description                                               Labels
+# --------------------------- --------------------------------------------------------- --------
+# my-cluster-mode-elasticache ElastiCache cluster in us-west-1 (configuration endpoint) ...
+# my-elasticache              ElastiCache cluster in us-west-1 (primary endpoint)       ...
+# my-elasticache-reader       ElastiCache cluster in us-west-1 (reader endpoint)        ...
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+$ tsh db ls
+# Name                        Description                                               Labels
+# --------------------------- --------------------------------------------------------- --------
+# my-cluster-mode-elasticache ElastiCache cluster in us-west-1 (configuration endpoint) ...
+# my-elasticache              ElastiCache cluster in us-west-1 (primary endpoint)       ...
+# my-elasticache-reader       ElastiCache cluster in us-west-1 (reader endpoint)        ...
+```
+
+</ScopedBlock>
+
+To connect to a particular database, first retrieve its certificate using the
+`tsh db login` command:
+
+```code
+$ tsh db login --db-user=my-database-user my-elasticache
+```
+
+If flag `--db-user` is not provided, Teleport logs in as the `default` user.
+
+<Admonition type="tip" title="Tip">
+  You can be logged in to multiple databases simultaneously.
+</Admonition>
+
+Now connect to the database:
```

Comment:
I think we need to connect the tab labeling scheme to the preceding instructions in this guide a bit more explicitly. Maybe add a sentence here to help the reader determine which tab to choose.